.devcontainer/Dockerfile

```diff
@@ -5,8 +5,14 @@ RUN apt-get update && apt-get install -y rustc cargo nodejs npm python3-sphinx p
 # Install required Sphinx extensions for the documentation
 RUN pip3 install furo sphinx-inline-tabs
 
+# Install required Sphinx extensions for the documentation
```

Comment:
I think there is a duplicated line here.

Reply:
Resolved.